src/common/SurgeStorage.h

```diff
@@ -178,6 +178,16 @@ enum NoiseColorChannels
     MONO = 1
 };
 
+enum CxorMode
+{
+    RINGMOD = 0,
```

Comment:
Also we generally use all lowercase for enums. You'll want these something like:

rmm_default,
rmm_cxor1,
rmm_cxor2,
etc.

Reply:
ah i wasn't sure, tried to make it fit with the others